packages/artifacts/junit-integration-tests.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="jest tests" tests="0" failures="0" errors="0" time="0.527">
+</testsuites>
```

Comment:
@stephenh not sure why this was created. Does this look correct?

Reply:
Hm, no, not sure where this came from... The `junit...xml` files are only used by CircleCI to do test reporting.